bug-reporting.md

```diff
@@ -51,6 +51,33 @@ In general, the following template should suffice for reporting a bug.
 
 ---
 
+## Raising feedback in Jira
+
+> General guidance for how feedback should be raised whilst working on KOMODO-managed Jira boards
```

Comment:
Do we want to add a note in here where if the team agrees that a particular piece of feedback doesn't need to be addressed as part of the parent ticket, that it could be converted to a normal task and moved to the backlog to pick up later? So that it's not lost but also doesn't block the completion of the parent?

Reply:
Absolutely. Could run the process as:

- QA raise child ticket
- QA add descriptive labels, uses the new labels added here
- QA add severity, uses Kyle's severity levels
- PM prioritises, converts tickets if needed
- Repeat as necessary

To be expanded of course